components/grid/export/events.md

```diff
@@ -367,18 +504,101 @@ The `OnAfterExport` event fires after [OnBeforeExport](#onbeforeexport) and befo
 </TelerikGrid>
 
 @code {
+    private MemoryStream csvStream { get; set; }
+
+    private IEnumerable<object> SelectedItems = Enumerable.Empty<object>();
+
+    private List<SampleData> GridData { get; set; }
+    
+    private bool ExportAllPages { get; set; }
+
     private async Task OnCSVAfterExport(GridAfterCsvExportEventArgs args)
     {
         var bytes = args.Stream.ToArray();
         var excelStream = new MemoryStream(bytes);       
     }
 
-    private MemoryStream csvStream { get; set; }
+    protected override void OnInitialized()
+    {
+        GridData = Enumerable.Range(1, 100).Select(x => new SampleData
+            {
+                ProductId = x,
+                ProductName = $"Product {x}",
+                UnitsInStock = x * 2,
+                Price = 3.14159m * x,
+                Discontinued = x % 4 == 0,
+                FirstReleaseDate = DateTime.Now.AddDays(-x)
+            }).ToList();
+    }
+
+    public class SampleData
+    {
+        public int ProductId { get; set; }
+        public string ProductName { get; set; }
+        public int UnitsInStock { get; set; }
+        public decimal Price { get; set; }
+        public bool Discontinued { get; set; }
+        public DateTime FirstReleaseDate { get; set; }
+    }
+}
+````
+
+### For PDF Export
+
+* `Stream` - `MemoryStream` - The output of the PDF export as a memory stream. The stream itself is finalized, so that the resource does not leak. To read and work with the stream, clone its available binary data to a new `MemoryStream` instance.
```

Comment:
Note that you're alternating between lowercase and capitals for the phrase after the dash. Stick to either but do not alternate between them. Also, note that our [Style Guide](https://docs.telerik.com/style-guide/punctuation) calls for `&mdash;` over a simple hyphen, especially for definition lists.